keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/CombinedRepositoryTests.kt

```diff
@@ -246,7 +259,7 @@ abstract class CombinedRepositoryTests<D : DeliveryConfigRepository, R : Resourc
           context("after an update") {
             before {
               resourcesDueForCheck()
-              subject.upsert(resource.copy(spec = DummyResourceSpec(id = resource.spec.id, data = "kthxbye")))
```

Comment:
I think some of the tests handling a single resource creation/update, without a delivery config associated with it and not relevant anymore.